### PR TITLE
have added the contact logos and made changes suggested by mike

### PR DESCRIPTION
**Pull request by mtmgoliath**
This should hopefully be all ready for main

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,14 +43,16 @@ <h2>About Me</h2>
     
     <section class="portfolioHighlightsContainer">
             <div class="portfolioHighlight">
-                <a href="#"></a>
-                <img class="portfolioImage" alt="" src="images/purplePink.jpg">
-                <h3>Project 1</h3>
+                <a href="#">
+                    <img class="portfolioImage" alt="" src="images/purplePink.jpg">
```
Comment (damienscarpellini) [inline index.html:47]: alt is empty

File: index.html [hunk 2]
```diff
@@ -43,14 +43,16 @@ <h2>About Me</h2>
...
+                    <h3>Project 1</h3>
+                </a>
             </div>
             <div class="portfolioHighlight">
-                <a href="#"></a>
-                <img class="portfolioImage" alt="" src="images/fireForest.jpg">
-                <h3>Project2</h3>
+                <a href="#">
+                    <img class="portfolioImage" alt="" src="images/fireForest.jpg">
```
Comment (damienscarpellini) [inline index.html:53]: alt is empty

File: index.html [hunk 3]
```diff
@@ -61,26 +63,30 @@ <h2>Portfolio</h2>
...
     </section>
     
-    <section class="portfolioHighlightsContainer">
+    <section class="portfolioHighlightsContainer projects1n2">
             <div class="portfolioHighlight">
-                <a href="#"></a>
-                <img class="portfolioImage" alt="" src="images/sworls.jpg">
-                <h3>Project 3</h3>
+                <a href="#">
+                    <img class="portfolioImage" alt="" src="images/sworls.jpg">
```
Comment (damienscarpellini) [inline index.html:69]: alt empty

File: index.html [hunk 4]
```diff
@@ -61,26 +63,30 @@ <h2>Portfolio</h2>
...
+                    <h3>Project 3</h3>
+                </a>
             </div>
             <div class="portfolioHighlight">
-                <a href="#"></a>
-                <img class="portfolioImage" alt="" src="images/purplePink.jpg">
-                <h3>Project 4</h3>
+                <a href="#">
+                    <img class="portfolioImage" alt="" src="images/purplePink.jpg">
```
Comment (damienscarpellini) [inline index.html:75]: alt empty

File: index.html [hunk 5]
```diff
@@ -61,26 +63,30 @@ <h2>Portfolio</h2>
...
+                    <h3>Project 4</h3>
+                </a>
             </div>
             <div class="portfolioHighlight">
-                <a href="#"></a>
-                <img class="portfolioImage" alt="" src="images/fireForest.jpg">
-                <h3>Project 5</h3>
+                <a href="#">
+                    <img class="portfolioImage" alt="" src="images/fireForest.jpg">
```
Comment (damienscarpellini) [inline index.html:81]: alt empty

File: style.css [hunk 6]
```diff
@@ -108,18 +107,17 @@ h2 {
...
+    justify-content: space-between;
+    margin: 4rem 6rem 2rem 6rem;
+    overflow: hidden;
 }
 
-.portfolioHighlight {
-    margin: 0rem 1rem 0rem 1rem;
-    text-align: center;
+.projects1n2 {
+    justify-content: space-evenly
```
Comment (damienscarpellini) [inline style.css]: missing semi colon

File: style.css [hunk 7]
```diff
@@ -226,6 +241,10 @@ footer {
     .aboutMe, .portfolioBlurb, .contactSection {
         margin: 0rem;
     }
+
+    /* .portfolioHighlightsContainer {
```
Comment (damienscarpellini) [inline style.css]: code commented out

**Review by damienscarpellini — APPROVED** [review]

File: index.html (hunk 5 above)
Comment (mtmgoliath) [reply index.html:81]: standard practice for decorative images

**Pull request merged**